pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -463,6 +461,10 @@ func (s *Service) SearchUsersPermissions(ctx context.Context, usr identity.Reque
 		return nil, err
 	}
 
+	if s.features.IsEnabled(ctx, featuremgmt.FlagAccessActionSets) {
+		options.ActionSets = s.actionResolver.ResolveActionPrefix(options.Action)
```

Comment:
Should not it be something like this (and same below): 
```suggestion
		options.ActionSets = s.actionResolver.ResolveAction(options.Action)
		options.ActionSets = append(options.ActionSets, 
		    s.actionResolver.ResolveActionPrefix(options.ActionPrefix))
```

Reply:
You're right. Clearly, I also wan't thinking straight anymore 🙈 Thanks for correcting!

Reply:
superrrnit: 

could we potentially run into not calling the correct method in the future. 

I have been trying to come up with a better way but the only renaming i could think of is to add `By`Prefix, `By`Name

Reply:
`ResolveActionByName`
`ResolveActionByPrefix`